package.json

```diff
@@ -30,7 +30,9 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "adb": "adb reverse tcp:9090 tcp:9090 && adb reverse tcp:3000 tcp:3000 && adb reverse tcp:9001 tcp:9001 && adb reverse tcp:8081 tcp:8081",
-    "postinstall": "node ./bin/postInstall"
+    "postinstall": "node ./bin/postInstall",
+    "webflow-codegen": "tsx ./scripts/webflow-codegen.ts",
+    "webflow-codegen-test": "tsx ./scripts/webflow-codegen-test.ts"
```

Comment:
You can run these two scripts to generate Webflow API types and test whether they are valid against real data